public/config.json

```diff
@@ -376,6 +377,26 @@
                 ]},
                 0
             ]},
+            { "if": [
+                { "===": [
+                    { "var": "payment_type" },
+                    "Check"
+                ]},
+                {"reduce": [
```

Comment:
btw the reason this is under a separate reduce is because jsonlogic destroys lexical scope inside reduces, which is dumb.